server/project/models/participant.py

```diff
@@ -0,0 +1,24 @@
+from project import db
+
+
+class Participant(db.Model):
+    __tablename__ = 'participants'
+
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String(128), nullable=False)
+    email = db.Column(db.String(128), nullable=False)
```

Comment:
Should maybe require participants to have unque email? This wold require to lookup a participant before creating it when creating an apointment.

Reply:
This could be implemented if we do some changes to the models. Currently if I use a unique constraint if the participant then attempts to make a second appointment the request will fail since they already exist in the participants table. This can be fixed if we switch it to a many-to-many relationship though then we only need to store every participant once.

Reply:
Many-Many will also help support adding `User` to `Participant`, so we could automatically add appointments.

Reply:
Implemented.